working-groups/meta/README.md

```diff
@@ -0,0 +1,31 @@
+# Meta Working Group
+![working group status: active][status]
+
+This working group is dedicated to fleshout details of how the compiler team will organize itself.
+
+- **Leads:** [@nikomatsakis][nikomatsakis]
+- **Meeting Notes:** [All](NOTES.md)
+
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+
+## What is the goal of this working group?
+This working group aims to accomplish the following:
+
+- Organizing working groups
+- Organizing rustc calendar
+- Organizing videos and lectures
+- Organizing communication between members
+
+# How can I get involved?
+If you are interested in getting involved in this working group, you should try attend a meeting and
+introduce yourself or send a message in the Zulip stream. You can be added to the GitHub and Zulip
+group for the working group if you are interested in being pinged when there are available tasks.
```

Comment:
Perhaps remove the last sentence about being added to the GitHub and Zulip groups as I'm not sure that this applies to the meta working group?

Reply:
We do have a Zulip user group

Reply:
🤦‍♂️